utils/test/testinput/gdas_icecamsr2ioda.yaml

```diff
@@ -1,12 +1,13 @@
-provider: AMSR2
-window begin: 2018-04-15T06:00:00Z
-window end: 2018-04-15T12:00:00Z
-output file: icec_amsr2_north.ioda.nc
-input files:
-- icec_amsr2_north_1.nc4
-- icec_amsr2_north_2.nc4
-
-test:
-  reference filename: testref/icecamsr2ioda.test
-  test output filename: testoutput/icecamsr2ioda.test
-  float relative tolerance: 1e-6
+provider: AMSR2
+window begin: 2018-04-15T06:00:00Z
+window end: 2018-04-15T12:00:00Z
+output file: icec_amsr2_north.ioda.nc
+#ocean basin: RECCAP2_region_masks_all_v20221025.nc
```

Comment:
Is there a purpose to have the commented line here?

Reply:
> Is there a purpose to have the commented line here?

Like a previous [PR#732](https://github.com/NOAA-EMC/GDASApp/pull/732), I commented out ocean basin as default.

Reply:
Yes @ShastriPaturi , it's a good reminder that it's not part of the ctest, and needs to be used when processing obs.